### ISSUE TITLE
Softmax and CrossEntropyLoss

### ISSUE THREAD

**Issue by gbuion** [nishnik/Deep-Semantic-Similarity-Model-PyTorch#2]
Thanks for the sharing.
I'm a little confused about Softmax and CrossEntropyLoss used in modeling.
As I know, the CrossEntropyLoss criterion do not need the model output parsed by Softmax.
So any guidance about this or just by mistake?
Thanks in advance!

**Comment by nishnik** [comment]
That was a mistake, thanks for pointing out.
